internal/characters/kazuha/burst.go

```diff
@@ -26,8 +26,9 @@ func init() {
 
 func (c *char) Burst(p map[string]int) action.ActionInfo {
 
-	c.qInfuse = attributes.NoElement
-	c.infuseCheckLocation = combat.NewCircleHit(c.Core.Combat.Player(), 1.5, false, combat.TargettableEnemy, combat.TargettablePlayer, combat.TargettableGadget)
+	c.qAbsorb = attributes.NoElement
+	c.absorbCheckLocation = combat.NewCircleHit(c.Core.Combat.Player(), 1.5, false, combat.TargettableEnemy, combat.TargettablePlayer, combat.TargettableGadget)
```

Comment:
since both burst and skill uses this, wouldnt this cause problem if you do skill -> teleport -> burst?